.gitignore

```diff
@@ -57,6 +57,7 @@ derby.log
 hs_err_pid*
 replay_pid*
 .attach_pid*
+AutoValue_*.java
```

Comment:
I think this is already handled by them being generated under `/build/`?

Reply:
no, I had it in the source dir - not sure why